test/clients/clients_test.py

```diff
@@ -59,7 +51,7 @@ def _check_vagrant():
         for path in os.environ.get("PATH").split(":")
     )
     vagrant_plugins = subprocess.check_output([vagrant_command, "plugin", "list"])
-    return (vagrant, "vagrant-scp" in vagrant_plugins)
+    return (vagrant, "vagrant-scp" in str(vagrant_plugins))
```

Comment:
Should use `decode` instead of `str` here. It'll end up being a string repr of that `bytes` otherwise. e.g `"b'somebytes'"`

Reply:
That explains a lot!